### PR TITLE
cleanups

### PR DESCRIPTION
**Pull request by arnetheduck**
* avoid circular panda imports
* move deposit merkleization helpers to spec/
* normalize validator signature helpers to spec names / params
* remove redundant functions for remote signing

### CODE REVIEW

File: beacon_chain/nimbus_beacon_node.nim [hunk 1]
```diff
@@ -145,6 +144,22 @@ versionGauge.set(1, labelValues=[fullVersionStr, gitRevision])
 
 logScope: topics = "beacnde"
 
+func getPandas(stdoutKind: StdoutLogKind): VanityLogs =
```
Comment (tersec) [inline beacon_chain/nimbus_beacon_node.nim:147]: Keeping 🐼s isolated in one module (both the Unicode codepoints and the general idea) was intentional.

Comment (arnetheduck) [reply beacon_chain/nimbus_beacon_node.nim:147]: it also led to a circular import of `conf`, the way it was structured, which in turn led to a cascade of other imports in unrelated places - is it .. important that it stays there? if so , the logging output stuff needs to go into its own module

Comment (tersec) [reply beacon_chain/nimbus_beacon_node.nim:147]: Avoiding circular imports is more important, sure, and this seems basically fine.

Text editors should all be fairly good at Unicode by now, and if not, it'll show up in diffs, so maybe one of the concerns (to keep non-Latin-1-ish characters out of frequently modified files) isn't so crucial.

**Review by tersec — APPROVED** [review]

**Comment by tersec** [comment]
Jenkins CI failing because
```
./scripts/launch_local_testnet.sh: line 860: EL_RPC_PORTS: unbound variable
script returned exit code 1
```
but this PR doesn't modify any of those scripts, so it's something pre-existing.

**Comment by etan-status** [comment]
> Jenkins CI failing because
> 
> ```
> ./scripts/launch_local_testnet.sh: line 860: EL_RPC_PORTS: unbound variable
> script returned exit code 1
> ```
> 
> but this PR doesn't modify any of those scripts, so it's something pre-existing.

need to merge from `unstable`, there was some regression that seems to have been fixed there.

**Comment by tersec** [comment]
Not sure it is, see https://github.com/status-im/nimbus-eth2/pull/3822

https://github.com/status-im/nimbus-eth2/commit/a48d741022f6b0da1bb679e0ede4e38c019242cf caused the `EL_RPC_PORTS` error, then https://github.com/status-im/nimbus-eth2/commit/73227508db6f8ece8657f33c76d2762691ac172e fixed that by defining said variable, but didn't fix the JWT setup.

Maybe one of the commits since then picked that up.

**Comment by tersec** [comment]
At least through https://ci.status.im/blue/organizations/jenkins/nimbus%2Fnimbus-eth2%2Fmultibranch/detail/unstable/1460/pipeline/149 (https://github.com/status-im/nimbus-eth2/pull/3796) it's still failing with:
```
{"lvl":"FAT","ts":"2022-06-28 23:04:24.832+00:00","msg":"Specified a JWT secret file which couldn't be loaded","topics":"beacnde","err":"couldn't open specified JWT secret file"}
```

https://github.com/status-im/nimbus-eth2/pull/3822 should fix this though.

**Pull request merged**